migration/servicedmigration/service.py

```diff
@@ -0,0 +1,28 @@
+
+import sys
+import json
+
+from version import versioned
+
+
+class Service:
```

Comment:
We probably want some kind of doc for the SDK in general.  Can you add some pydoc comments for these methods as you go?